selfdrive/car/subaru/subarucan.py

```diff
@@ -76,6 +76,19 @@ def create_es_dashstatus(packer, dashstatus_msg):
 
   return packer.make_can_msg("ES_DashStatus", 0, values)
 
+def create_infotainmentstatus(packer, infotainmentstatus_msg, visual_alert):
+  values = copy.copy(infotainmentstatus_msg)
+
+  # Filter stock LKAS disabled and Keep hands on steering wheel OFF alerts
+  if values["LKAS_State_Infotainment"] in [3,4]:
+    values["LKAS_State_Infotainment"] = 0
+  
+  # Show Keep hands on wheel alert for openpilot steerRequired alert
```

Comment:
let's also show the obstacle detected alert when `visual_alert == VisualAlert.fcw`

Reply:
done. _needs verification_